MetroscopeModelingLibrary/MultiFluid/HeatExchangers/CoolingTower3.mo

```diff
@@ -130,8 +130,8 @@ equation
   Air_outlet.Q_out * (1 - Air_outlet.Xi_out[1]) = - Air_inlet.Q_in *(1 - Air_inlet.Xi_in[1]);
 
   Water_outlet.P_out = Water_inlet.P_in;
-  Water_outlet.T_out = Water_inlet.T_in;
-  Water_outlet.Q_out = Water_inlet.Q_in - Q_makeup;
+  //Water_outlet.T_out = Water_inlet.T_in;
+  Water_outlet.Q_out = Q_hot - Q_makeup;
```

Comment:
Q_out it negative (flow going out a componenet is negative by convention, so here it should be :
`Water_outlet.Q_out = - (Q_hot - Q_makeup);`

Reply:
I think it could be a could idea to create a new variable `Q_hot_out `and renaming `Q_hot `by `Q_hot_in`